<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Using HumanIslets.com | Human Islets</title>
  <meta name="description" content="This is an introduction of the Human Islet project." />
  <meta name="generator" content="bookdown 0.39 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Using HumanIslets.com | Human Islets" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="This is an introduction of the Human Islet project." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Using HumanIslets.com | Human Islets" />
  
  <meta name="twitter:description" content="This is an introduction of the Human Islet project." />
  

<meta name="author" content="" />


<meta name="date" content="2024-05-30" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="overview-of-primary-metadata.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Human Islets</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> General Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#land-acknowledgement"><i class="fa fa-check"></i><b>1.1</b> Land Acknowledgement</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#terms-of-use"><i class="fa fa-check"></i><b>1.2</b> Terms of Use</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#financial-support"><i class="fa fa-check"></i><b>1.3</b> Financial Support</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#overview-and-sampledata-collection"><i class="fa fa-check"></i><b>1.4</b> Overview and Sample/data Collection</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="using-humanislets.html"><a href="using-humanislets.html"><i class="fa fa-check"></i><b>2</b> Using HumanIslets.com</a>
<ul>
<li class="chapter" data-level="2.1" data-path="using-humanislets.html"><a href="using-humanislets.html#omics-data-analysis"><i class="fa fa-check"></i><b>2.1</b> Omics Data Analysis <img src="logo1.png" alt="Logo" style="width:18px; height:18px;"></a></li>
<li class="chapter" data-level="2.2" data-path="using-humanislets.html"><a href="using-humanislets.html#donor-view"><i class="fa fa-check"></i><b>2.2</b> Donor View <img src="logo2.png" alt="Logo" style="width:18px; height:18px;"></a></li>
<li class="chapter" data-level="2.3" data-path="using-humanislets.html"><a href="using-humanislets.html#feature-view"><i class="fa fa-check"></i><b>2.3</b> Feature View <img src="logo3.png" alt="Logo" style="width:18px; height:18px;"></a></li>
<li class="chapter" data-level="2.4" data-path="using-humanislets.html"><a href="using-humanislets.html#data-download"><i class="fa fa-check"></i><b>2.4</b> Data download <img src="logo4.png" alt="Logo" style="width:18px; height:18px;"></a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html"><i class="fa fa-check"></i><b>3</b> Overview of Primary Metadata</a>
<ul>
<li class="chapter" data-level="3.1" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#donor-characteristics"><i class="fa fa-check"></i><b>3.1</b> Donor Characteristics</a></li>
<li class="chapter" data-level="3.2" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#organ-characteristics-and-processing"><i class="fa fa-check"></i><b>3.2</b> Organ Characteristics and Processing</a></li>
<li class="chapter" data-level="3.3" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#isolation-outcomes"><i class="fa fa-check"></i><b>3.3</b> Isolation Outcomes</a></li>
<li class="chapter" data-level="3.4" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#cell-type-proportions"><i class="fa fa-check"></i><b>3.4</b> Cell Type Proportions</a></li>
<li class="chapter" data-level="3.5" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#cell-culture-outcomes"><i class="fa fa-check"></i><b>3.5</b> Cell Culture Outcomes</a></li>
<li class="chapter" data-level="3.6" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#static-insulin-secretion"><i class="fa fa-check"></i><b>3.6</b> Static Insulin Secretion</a></li>
<li class="chapter" data-level="3.7" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#islet-oxygen-consumption-seahorse-assay"><i class="fa fa-check"></i><b>3.7</b> Islet Oxygen Consumption (Seahorse assay)</a></li>
<li class="chapter" data-level="3.8" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#dynamic-insulin-responses-to-macronutrients"><i class="fa fa-check"></i><b>3.8</b> Dynamic Insulin Responses to Macronutrients</a></li>
<li class="chapter" data-level="3.9" data-path="overview-of-primary-metadata.html"><a href="overview-of-primary-metadata.html#single-cell-function-electrophysiology"><i class="fa fa-check"></i><b>3.9</b> Single-cell Function (electrophysiology)</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="overview-of-omics-data-types.html"><a href="overview-of-omics-data-types.html"><i class="fa fa-check"></i><b>4</b> Overview of Omics Data Types</a>
<ul>
<li class="chapter" data-level="4.1" data-path="overview-of-omics-data-types.html"><a href="overview-of-omics-data-types.html#gene-expression-bulk-rna-seq"><i class="fa fa-check"></i><b>4.1</b> Gene expression (bulk RNA-seq)</a></li>
<li class="chapter" data-level="4.2" data-path="overview-of-omics-data-types.html"><a href="overview-of-omics-data-types.html#gene-expression-nanostring"><i class="fa fa-check"></i><b>4.2</b> Gene expression (Nanostring)</a></li>
<li class="chapter" data-level="4.3" data-path="overview-of-omics-data-types.html"><a href="overview-of-omics-data-types.html#protein-expression-proteomics"><i class="fa fa-check"></i><b>4.3</b> Protein expression (proteomics)</a></li>
<li class="chapter" data-level="4.4" data-path="overview-of-omics-data-types.html"><a href="overview-of-omics-data-types.html#gene-expression-pseudobulk-rna-seq"><i class="fa fa-check"></i><b>4.4</b> Gene expression (pseudobulk RNA-seq)</a></li>
<li class="chapter" data-level="4.5" data-path="overview-of-omics-data-types.html"><a href="overview-of-omics-data-types.html#single-cell-gene-expression-patch-seq"><i class="fa fa-check"></i><b>4.5</b> Single-cell gene expression (patch-seq)</a></li>
<li class="chapter" data-level="4.6" data-path="overview-of-omics-data-types.html"><a href="overview-of-omics-data-types.html#single-cell-rna-seq"><i class="fa fa-check"></i><b>4.6</b> Single-cell RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="overview-of-statistical-analysis-methods.html"><a href="overview-of-statistical-analysis-methods.html"><i class="fa fa-check"></i><b>5</b> Overview of statistical analysis methods</a>
<ul>
<li class="chapter" data-level="5.1" data-path="overview-of-statistical-analysis-methods.html"><a href="overview-of-statistical-analysis-methods.html#multiple-linear-regression"><i class="fa fa-check"></i><b>5.1</b> Multiple linear regression</a></li>
<li class="chapter" data-level="5.2" data-path="overview-of-statistical-analysis-methods.html"><a href="overview-of-statistical-analysis-methods.html#spearman-ranked-correlation"><i class="fa fa-check"></i><b>5.2</b> Spearman ranked correlation</a></li>
<li class="chapter" data-level="5.3" data-path="overview-of-statistical-analysis-methods.html"><a href="overview-of-statistical-analysis-methods.html#pathway-analysis"><i class="fa fa-check"></i><b>5.3</b> Pathway analysis</a></li>
<li class="chapter" data-level="5.4" data-path="overview-of-statistical-analysis-methods.html"><a href="overview-of-statistical-analysis-methods.html#pre-computed-feature-associations"><i class="fa fa-check"></i><b>5.4</b> Pre-computed feature associations</a></li>
<li class="chapter" data-level="5.5" data-path="overview-of-statistical-analysis-methods.html"><a href="overview-of-statistical-analysis-methods.html#deconvolution-analysis"><i class="fa fa-check"></i><b>5.5</b> Deconvolution analysis</a></li>
<li class="chapter" data-level="5.6" data-path="overview-of-statistical-analysis-methods.html"><a href="overview-of-statistical-analysis-methods.html#sources-of-gene-annotations-link-to-code-repository"><i class="fa fa-check"></i><b>5.6</b> Sources of gene annotations &amp; link to code repository</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Human Islets</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="using-humanislets.com" class="section level1 hasAnchor" number="2">
<h1><span class="header-section-number">Chapter 2</span> Using HumanIslets.com<a href="using-humanislets.html#using-humanislets.com" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="omics-data-analysis" class="section level2 hasAnchor" number="2.1">
<h2><span class="header-section-number">2.1</span> Omics Data Analysis <a href="https://humanislets.com"><img src="logo1.png" alt="Logo" style="width:18px; height:18px;"></a><a href="using-humanislets.html#omics-data-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>HumanIslet.com presently includes information from 540 human research islet isolations that includes donor and technical metadata, islet and islet cell function, and omics. Omics datasets include bulk proteomics, bulk RNAseq, gene panels (Nanostring), patch-seq, and pseudobulk RNAseq for alpha- and beta-cells. Omics-metadata relationships can be analysed by:</p>
<ol style="list-style-type: decimal">
<li>Selecting the ‘Omics Type’ of interest</li>
<li>Selecting the ‘Primary Metadata’ to be analysed, and ‘Comparison of Interest’ if applicable</li>
<li>Choosing covariates to control for</li>
<li>Selecting donors to include
<ol style="list-style-type: lower-alpha">
<li>Choosing ‘Subset’ will open a window that allows for filtering by donor characteristics and data availability, or upload of a list of donor IDs or RRIDs of interest</li>
</ol></li>
<li>Selecting a p-value cutoff, and the use of adjusted p-values</li>
<li>Click ‘submit’</li>
</ol>
<p>The number of donors included in the analysis will be displayed. If too few donors are available for the chosen analysis (&lt;10), an error message will be shown. Results of the analysis will be shown under ‘Feature Association Results’, and both results and analysis pipeline can be downloaded by clicking on the appropriate button. Results are shown as a graphical representation (volcano plot) or table, accessible by clicking on the appropriate tab. Clicking on features (points in the volcano plot, or table entries) will raise a graph showing categorical or scatterplot representations of the data/feature of interest. Clicking on individual donors within those plots will take you to the ‘Donor View’ page for that donor (section 2.2, below), allowing for inspection of potential outlier donors. In the Results Table, clicking on the feature name will take you to the ‘Feature Search’ page (section 2.3, below) for that gene/protein. At the right are links to the NCBI gene page and T2D Knowledge Portal (T2DKP) entries for each feature (external sites), and to a single-cell RNAseq reference dataset (SC) to inspect cell-type specific expression of the feature.</p>
<p>The third tab under the ‘Feature Association Results’ pulls up a Pathway Analysis tool for common Overrepresentation Analysis (ORA) or Gene Set Enrichment Analysis (GSEA) on the results. To perform pathway analysis:</p>
<ol style="list-style-type: decimal">
<li>Select ORA or GSEA
<ol style="list-style-type: lower-alpha">
<li>ORA is performed on significant hits based on the p-value threshold set above</li>
<li>GSEA can be performed using log2FC/coefficient or T-statistic for ranking, and can be performed even in cases when no individual features are marked as significant</li>
</ol></li>
<li>Select whether to collapse redundant pathways</li>
<li>Choose a false discovery rate (FDR) cutoff</li>
<li>Choose among common pathway libraries to query</li>
<li>Click ‘submit’</li>
</ol>
<p>The ‘Pathway Results’ will be displayed in both graphical and table format, selected by clicking on the appropriate tab. As above, both results and analysis pipeline can be downloaded by clicking on the appropriate button. In the interactive graphical view, significant pathways (based on FDR cutoff) are shown as ridgeplots. Clicking on these will raise heatmaps showing the contribution of individual genes/proteins.</p>
</div>
<div id="donor-view" class="section level2 hasAnchor" number="2.2">
<h2><span class="header-section-number">2.2</span> Donor View <a href="https://humanislets.com"><img src="logo2.png" alt="Logo" style="width:18px; height:18px;"></a><a href="using-humanislets.html#donor-view" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The Donor View page allows examination of individual donor, islet isolation, and functional data from each of 540 donors currently in the HumanIslets.com database. Donors can be searched by ADI IsletCore Donor ID or Research Resource ID (RRID). Data categories can be selected in tabs indicated on the left (Donor information; Isolation Information; Static and Dynamic Insulin Secretion; Mitochondrial Function; Electrophysiology, along with an overview of Omics data currently available). These metadata are described in detail in chapters 3 and 4, below.</p>
<p>Typically, data is shown for selected donor in the context of the entire dataset, allowing for easy inspection and determination of whether a donor falls within the normal range of the population. A few points that are worth noting here:</p>
<ul>
<li><em>Donor Information</em> shows the RRID if available. Clicking on this will link out to the NCBI Biosample page for this donor</li>
<li><em>Isolation Information</em> also contains the current inventory of snap frozen and cryopreserved islet tubes for that donor (all donors also have paraffin embedded biopsies)</li>
<li><em>Isolation Information</em> includes cell type composition calculated by - deconvolution in donors with bulk proteomics data (see section 5.5, below)</li>
<li><em>Dynamic Insulin</em> Responses can be shown with or without normalisation to baseline</li>
<li><em>Mitochondrial Function</em> (oxygen consumption by Seahorse assay) can be shown with or without normalisation to DNA and baseline</li>
</ul>
</div>
<div id="feature-view" class="section level2 hasAnchor" number="2.3">
<h2><span class="header-section-number">2.3</span> Feature View <a href="https://humanislets.com"><img src="logo3.png" alt="Logo" style="width:18px; height:18px;"></a><a href="using-humanislets.html#feature-view" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Instead of performing an ‘Omics Analysis’ focused on a particular omics type and metadata variable, users can search for a feature of interest and pull up ALL associations for that particular feature of interest, corrected for common covariates (see section 5.4, below). Searchable features include genes/proteins and also metadata variable. This allows rapid interrogation of precomputed results across all datasets.</p>
<p>Searching for a gene/protein feature generates a table of results ranked by adjusted p-value. Searching for a metadata variable (e.g. BMI) generates a table of feature (gene/protein) results across all omics datasets, ranked by adjusted p-value. Clicking on rows and external links brings up categorical or scatterplots of the association, external pages for the feature of interest, or single-cell RNAseq expression for that feature. Because a large number of associations can be displayed, filtering of the metadata or omics type is enabled by clicking on the column title.</p>
</div>
<div id="data-download" class="section level2 hasAnchor" number="2.4">
<h2><span class="header-section-number">2.4</span> Data download <a href="https://humanislets.com"><img src="logo4.png" alt="Logo" style="width:18px; height:18px;"></a><a href="using-humanislets.html#data-download" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>All data at HumanIslets.com is available for download. Donors of interest can be filtered by Donor Characteristics (see section 3.1) and/or data availability. Advanced filters allow selection of donors of interest by any metadata available. Donors of interest can also be selected by uploading a list of donor IDs (for example, donor IDs obtained from a particular Omics Analysis). Data can be downloaded as Comma separated values (.csv), text (.txt), R object (.rds), or in Excel format (.xlsx) within a zip file. To perform data download:</p>
<ol style="list-style-type: decimal">
<li>Under ‘Select Donors’ choose filtering parameters, upload a list of donor IDs
<ol style="list-style-type: lower-alpha">
<li>Distribution of chosen donors compared with the entire dataset can be assessed by clicking on the icons at right</li>
</ol></li>
<li>Once donors of interest are selected/filtered, click ‘submit’ to choose those donors
<ol style="list-style-type: lower-alpha">
<li>The total number of donors chosen will be displayed</li>
</ol></li>
<li>Under ‘Download Data’
<ol style="list-style-type: lower-alpha">
<li>Select the datasets of interest</li>
<li>Select the data format</li>
<li>The distribution of datasets selected, compared with the entire database, can be shown by clicking ‘Summary’</li>
</ol></li>
<li>Click ‘Download’</li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="overview-of-primary-metadata.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://https://github.com/xia-lab/HumanIsletsDoc/blob/main/01-using-website.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
